### DISCUSSION TITLE
Incomplete Data

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#55] [General]
INSPIRE data models are designed to collect a large amount of information, but in many cases the attributes are not mandatory.

This lack of obligation makes it difficult to work in a cross-border context, as although the same data model is used, depending on the provider, some data may be incomplete.

It is important that each provider provide all the information they have, even if it is not mandatory, and strive to encode it according to INSPIRE so that the data is more useful and beneficial for institutions and citizens.

**Comment by thorsten-reitz** [comment]
Here we should clarify what the barrier/issue is. As such, it reads like a recommendation.

**Comment by AndersOstmanNovogit** [comment]
This issue is covered in #38 and #39. It is then reported in section 4.2.3 of D4.3.
